docs/04-azure-requirements.md

```diff
@@ -0,0 +1,30 @@
+## Azure Requirements
+
+To run your R code across a cluster in Azure, we'll need to get keys and account information.
+
+### Setup Azure Account
+First, set up your Azure Account ([Get started for free!](https://azure.microsoft.com/en-us/free/))
+
+Once you have an Azure account, you'll need to create the following two services in the Azure portal:
+  - Azure Batch Account ([Create an Azure Batch Account in the Portal](https://docs.microsoft.com/en-us/azure/Batch/batch-account-create-portal))
```

Comment:
Azure Batch Account line and Storage Account line don't have the same indentation

Reply:
Updated